modules/Expectation.js

```diff
@@ -333,10 +333,21 @@ class Expectation {
     const expectedArgs = Array.prototype.slice.call(arguments, 0)
 
     assert(
+        spy.calls.length > 0,
+        ('spy was not called')
+    )
+
+
+    let actualArguments = ''
+    for(const arg of spy.calls) {
+      actualArguments =  actualArguments + '- ' + JSON.stringify(arg.arguments) + '\n'
+    }
+    const message = 'spy was never called with:\n%s.\n\nIt was called with:\n' + actualArguments
```

Comment:
I tried using the `%s` place holder instead of the concatenation here, but it did not interpreted the `\n`, outputting it literally without breaking the line.